deploy/helm/kubecf/templates/brain-tests.yaml

```diff
@@ -0,0 +1,223 @@
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - "services"
+  verbs:
+  - "create"
+  - "get"
+  - "delete"
+  - "list"
+  - "patch"
+- apiGroups:
+  - "extensions"
+  resourceNames:
+  - "default"
+  resources:
+  - "podsecuritypolicies"
+  verbs:
+  - "use"
+- apiGroups:
+  - "extensions"
+  resources:
+  - "replicasets"
+  verbs:
+  - "create"
+  - "get"
+  - "list"
+  - "delete"
+  - "update"
+- apiGroups:
+  - "apps"
+  resources:
+  - "statefulsets"
+  verbs:
+  - "create"
+  - "get"
+  - "update"
+  - "delete"
+  - "list"
+  - "patch"
+- apiGroups:
+  - ""
+  resources:
+  - "pods"
+  verbs:
+  - "create"
+  - "get"
+  - "list"
+  - "delete"
+  - "update"
+  - "patch"
+- apiGroups:
+  - ""
+  resources:
+  - "pods/exec"
+  verbs:
+  - "create"
+  - "delete"
+- apiGroups:
+  - ""
+  resources:
+  - "pods/log"
+  verbs:
+  - "create"
+  - "delete"
+  - "get"
+  - "list"
+- apiGroups:
+  - ""
+  resources:
+  - "secrets"
+  verbs:
+  - "create"
+  - "get"
+  - "delete"
+  - "list"
+
+# Role binding for service account "tests-brain" and role "test-role-brain"
+---
+apiVersion: "rbac.authorization.k8s.io/v1"
+kind: "RoleBinding"
+metadata:
+  name: "{{ .Values.deployment_name }}-tests-brain-test-role-brain-binding"
+  labels:
+    app.kubernetes.io/component: "{{ .Values.deployment_name }}-tests-brain-test-role-brain-binding"
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io/name: {{ include "kubecf.fullname" . }}
+    app.kubernetes.io/version: {{ default .Chart.Version .Chart.AppVersion | quote }}
+    helm.sh/chart: {{ include "kubecf.chart" . }}
+subjects:
+- kind: "ServiceAccount"
+  name: "{{ .Values.deployment_name }}-tests-brain"
+roleRef:
+  apiGroup: "rbac.authorization.k8s.io"
+  kind: "Role"
+  name: "{{ .Values.deployment_name }}-test-role-brain"
+
+# Cluster role "test-cluster-role" only used by account "tests-brain"
+---
+apiVersion: "rbac.authorization.k8s.io/v1"
+kind: "ClusterRole"
+metadata:
+  name: "{{ .Release.Namespace }}-{{ .Values.deployment_name }}-test-cluster-role"
+  labels:
+    app.kubernetes.io/component: "{{ .Release.Namespace }}-{{ .Values.deployment_name }}-test-cluster-role"
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io/name: {{ include "kubecf.fullname" . }}
+    app.kubernetes.io/version: {{ default .Chart.Version .Chart.AppVersion | quote }}
+    helm.sh/chart: {{ include "kubecf.chart" . }}
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - "namespaces"
+  verbs:
+  - "create"
+  - "get"
+  - "delete"
+- apiGroups:
+  - ""
+  resources:
+  - "pods"
+  - "pods/log"
+  - "namespaces"
+  verbs:
+  - "get"
+  - "list"
+  - "watch"
+- apiGroups:
+  - ""
+  resources:
+  - "pods/portforward"
+  verbs:
+  - "create"
+- apiGroups:
+  - "rbac.authorization.k8s.io"
+  resources:
+  - "clusterrolebindings"
```

Comment:
This is to [delete the minibroker cluster role binding](https://github.com/SUSE/brain-tests-release/blob/b291ee8f850052e940801260d7a85817cbf9692a/src/acceptance-tests-brain/test-scripts/minibroker_helper.rb#L99) that we… never created? Huh.

Reply:
Thus review after the tests have been made to pass. Maybe then it will be needed. And if not, it goes.